locales/en-US/strings.properties

```diff
@@ -171,3 +171,24 @@ section_menu_action_add_topsite=Add Top Site
 section_menu_action_move_up=Move Up
 section_menu_action_move_down=Move Down
 section_menu_action_privacy_notice=Privacy Notice
+
+# LOCALIZATION NOTE (firstrun_*). These strings are displayed only once, on the
+# firstrun of the browser, they give an introduction to Firefox and Sync.
+firstrun_title=Take Firefox with You
+firstrun_content=Get your bookmarks, history, passwords and other settings on all your devices.
+firstrun_learn_more_link=Learn more about Firefox Accounts
+
+# LOCALIZATION NOTE (firstrun_form_header and firstrun_form_sub_header):
+# firstrun_form_sub_header is a continuation of firstrun_form_header, they are one sentance.
+# firstrun_form_header is displayed more boldly as the call to action.
+firstrun_form_header=Enter your email
+firstrun_form_sub_header=to continue to Firefox Sync.
+
+# LOCALIZATION NOTE (firstrun_extra_legal_links): {terms} is equal to firstrun_terms_of_service, and
+# {privacy} is equal to firstrun_privacy_notice. {terms} and {privacy} are clickable links.
+firstrun_extra_legal_links=By proceding you agree to the {terms} and {privacy}.
```

Comment:
typo: `proceding` -> `proceeding`

FxA also has a comma after the word: 
<img src=https://user-images.githubusercontent.com/128755/38882548-62dcb0ce-4238-11e8-9aac-cb34cdf3bd34.png width=400 /> 

Not sure if you want that here

Reply:
yeah, I want it as identical as possible